docs/reference/custom_fields/fields_list.md

```diff
@@ -0,0 +1,98 @@
+# Fields list
+
+_Introduced in InvenioRDM v12_
+
+The following document is a reference guide for all the custom fields implemented and distributed in invenio.
+
+!!!tip "How to add a field to an instance?"
+    If you want to add a field to an instance, please refer to the custom fields [documentation](../../customize/metadata/custom_fields/records.md#add-reusable-custom-fields-to-records).
+
+## Publishing information
+
+This group of fields contains the metadata for "Journal", "Imprint" and "Thesis". Each field has its own data representation, however they are grouped in the UI.
+
+**Deposit form** 
+
+![Deposit form](../../images/pub_info_deposit_form.png)
+
+**Landing page**
+
+![Landing page](../../images/pub_info_landing_page.png)
+
+Publishing information is displayed in the section "Details", under "Published in".
+
+### Journal
+
+This nested field implements a journal's metadata. It can be used to describe a journal where an article was published.
```

Comment:
```suggestion
This field implements a journal's metadata. It can be used to describe a journal where an article was published.
```

I'm not sure there is added value on exposing how it is implemted and can be confusing. Same happens in other places in this file.